src/AppInstallerCLICore/Public/ConfigurationSetProcessorFactoryRemoting.h

```diff
@@ -35,6 +35,9 @@ namespace AppInstaller::CLI::ConfigurationRemoting
         // The path to the dsc.exe executable, as discovered.
         // Read only.
         FoundDscExecutablePath,
+        // Whether to request detailed traces from the processor.
+        // Read / Write
+        DiagnosticTraceLevel,

```

Comment:
nit: the name "*TraceLevel" makes me think it's taking a trace level setting like warning, verbose while after reading the code, it's more like "DiagnosticTraceEnabled"

Reply:
I will do that first thing in my next change so I don't need to wait for another build cycle.